source/usage-examples/updateMany.txt

```diff
@@ -7,14 +7,17 @@ Update Multiple Documents
 You can update multiple documents in a collection by using the ``UpdateMany()``
 method.
 
-The following example passes a query filter and an update parameter to the
-``UpdateMany()`` method, which matches documents in the
-``listingsAndReviews`` collection where the value of the ``address.market``
-field is "Sydney" and multiplies the ``price`` field in the matching
-documents by a factor of *1.15*:
+Example
+-------
 
 .. include:: /includes/usage-examples/run-example-tip.rst
 
+The following example performs the following on the
+``listingsAndReviews`` collection:
+
+- Matches documents in which the ``address.market`` is "Sydney"
+- Multiplies the ``price`` in the matched documents by 1.15
```

Comment:
I think it could help the clarity of this bullet point to mention that it updates the document with the new prices.
Otherwise, it could be interpreted as just performing a multiplication operation and maybe printing that out instead of making any updates to the matched document(s).